configs/components/rubygem-ffi.rb

```diff
@@ -28,4 +28,44 @@
       "#{settings[:gem_install]} ffi-#{pkg.get_version}-#{platform.architecture}-mingw32.gem"
     end
   end
+
+  if platform.is_solaris? && platform.is_cross_compiled?
+    base_ruby = case platform.os_version
+                when "10"
+                  "/opt/csw/lib/ruby/2.0.0"
+                when "11"
+                  "/opt/pl-build-tools/lib/ruby/2.1.0"
+                end
+
+    case platform.os_version
+    when "10"
+      pkg.environment "PATH", "/opt/csw/bin:$$PATH"
+      pkg.install_file "/opt/csw/lib/libffi.so", "#{settings[:libdir]}/libffi.so.6"
```

Comment:
`ffi_c.so` is dynamically linked against `libffi.so`, so we need to include it in the runtime